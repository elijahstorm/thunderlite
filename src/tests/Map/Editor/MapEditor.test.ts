import { describe, it, expect, afterEach } from 'vitest'
import { cleanup, fireEvent, screen } from '@testing-library/svelte'

describe('MapRender.svelte', () => {
	afterEach(() => cleanup())

	it('mounts', () => {})

	it('updates on button click', async () => {})
})
