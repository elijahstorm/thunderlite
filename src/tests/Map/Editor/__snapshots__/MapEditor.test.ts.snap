// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MapRender.svelte > mounts 1`] = `"<div><div class=\\"flex gap-2 border-4 border-black h-full bg-stone-400\\"><p>loading...</p><!--<Game>--></div><!--<MapRender>--></div>"`;
